app/views/map/_peopleLeaflet.html.erb

```diff
@@ -12,17 +12,36 @@
 	<% end %>
 
   <script>
-	var map<%= unique_id %> = L.map('map<%= unique_id %>').on('load', onMapLoad).setView([<%= lat %>,<%= lon %>], 2);
- 	L.tileLayer("//a.tiles.mapbox.com/v3/jywarren.map-lmrwb2em/{z}/{x}/{y}.png").addTo(map<%= unique_id %>);
- 	function onMapLoad(e){
-     	$.getJSON("/api/srch/peoplelocations?srchString=100" , function(data){
- 			if (!!data.items){
- 				for (i = 0; i < data.items.length ; i++) { 
- 					var url = data.items[i].docUrl ;
- 					var title = data.items[i].docTitle ;
-   					L.marker([data.items[i].latitude , data.items[i].longitude]).addTo(map<%= unique_id %>).bindPopup("<a href=" +  url + ">" + title + "</a>") ;
-				}
- 			}
- 		});
-	}
+   var map<%= unique_id %> = L.map('map<%= unique_id %>').on('load', onMapLoad).setView([<%= lat %>,<%= lon %>], 2);
+   L.tileLayer("//a.tiles.mapbox.com/v3/jywarren.map-lmrwb2em/{z}/{x}/{y}.png").addTo(map<%= unique_id %>);
+   var searchLayer<%= unique_id %> = L.layerGroup().addTo(map<%= unique_id %>);
+    // add the search bar to the map
+   var controlSearch<%= unique_id %> = new L.Control.Search({
+      position:'topright',    // where do you want the search bar?
+      layer: searchLayer<%= unique_id %>,  // name of the layer
+      initial: false,
+      minLength: 1,
+      textErr: 'Person not found' , 
+      zoom: 11,        // set zoom to found location when searched
+      marker: false,
+      textPlaceholder: 'search people' // placeholder while nothing is searched
+    });
+   map<%= unique_id %>.addControl(controlSearch<%= unique_id %>);
+    /*
+    controlSearch<%= unique_id %>.on('search:collapsed', function(e) {
```

Comment:
So this is basically an Event Listener which is triggered when the search bar collapses .
Currently commented out .

Reply:
I have commented the zoom-out feature for now .